firebase-sessions/src/main/kotlin/com/google/firebase/sessions/api/FirebaseSessionsDependencies.kt

```diff
@@ -39,6 +39,19 @@ object FirebaseSessionsDependencies {
    * the Sessions SDK will never generate a session.
    */
   fun addDependency(subscriberName: SessionSubscriber.Name) {
+    if (subscriberName == SessionSubscriber.Name.PERFORMANCE) {
+      throw IllegalArgumentException(
+        """
+          Incompatible versions of Firebase Perf and Firebase Sessions.
+          A safe combination would be:
+            firebase-sessions:1.1.0
+            firebase-crashlytics:18.5.0
+            firebase-perf:20.5.0
```

Comment:
Do you want the message to reflect that a particular version of a library and above or just leave it at the specified version. Leaving it at a specified version is going to need to update this comment in the near future.

Reply:
I put this for now because I don't know what will happen in future versions. I expect we will make it compatible again, so maybe only this version will have this message. If you have a better idea for wording that reflects that we can work on the message.